lib/service.dart

```diff
@@ -76,7 +78,18 @@ const imgurClientId = 'Client-ID 2cfbc27ce77879d';
 const maxPing = 9007199254740991;
 
 class Service {
-  final log = Logger('Service');
+  final log;
+  final Client httpClient;
+
+  Service() :
+    log = Logger('Service'),
+    httpClient = Platform.isAndroid ?
+      CronetClient.fromCronetEngine(
+        CronetEngine.build(
+          cacheMode: CacheMode.memory,
```

Comment:
Do we really need caching ? I'm curious if it can be an issue with things that change often like comments

Reply:
I'll merge first but we can continue the conversation.